lerna.json

```diff
@@ -0,0 +1,25 @@
+{
+  "lerna": "2.0.0-beta.20",
+  "version": "6.9.0",
+  "changelog": {
+    "repo": "babel/babel",
+    "labels": {
+      "tag: bug fix": "Bug Fix",
+      "tag: breaking change": "Breaking Change",
+      "tag: documentation": "Documentation",
+      "tag: internal": "Internal",
+      "tag: new feature": "New Feature",
+      "tag: polish": "Polish",
+      "tag: spec compliancy": "Spec Compliancy"
+    }
+  },
+  "bootstrapConfig": {
+    "ignore": "babel-runtime"
```

Comment:
ignore `babel-runtime` which should replace the `rm -rf packages/*/node_modules/babel-runtime` above